emotesServer/src/main/java/io/github/kosmx/emotes/server/moderation/EmoteModerator.java

```diff
@@ -0,0 +1,44 @@
+package io.github.kosmx.emotes.server.moderation;
```

Comment:
You've made it so that this class is never initialized, if that's the intention, it's better to remove it